src/test/app/PermissionedDomains_test.cpp

```diff
@@ -77,6 +78,22 @@ class PermissionedDomains_test : public beast::unit_test::suite
         env(pdomain::deleteTx(alice, domain));
     }
 
+    // Verify that PD cannot be created or updated if credentials are disabled
+    void
+    testCredentialsDisabled()
+    {
+        auto amendments = supported_amendments();
+        amendments.set(featurePermissionedDomains);
+        amendments.reset(featureCredentials);
+        testcase("Credentials disabled");
+        Account const alice("alice");
+        Env env(*this, amendments);
+        env.fund(XRP(1000), alice);
+        pdomain::Credentials credentials{{alice, "first credential"}};
+        env(pdomain::setTx(alice, credentials), ter(temDISABLED));
+        env(pdomain::deleteTx(alice, uint256(75)), ter(tecNO_ENTRY));
```

Comment:
This test (`deleteTx`) doesn't interact with the feature at all, please remove it. 
It would be a good test if you would try to delete already **existent** permissioned domain  which was created before this feature. But I doubt such a test can be implemented withing unit tests (as it require restarting of the rippled and changing features set)

Reply:
Done, thanks